setup.cfg

```diff
@@ -0,0 +1,11 @@
+[metadata]
+name=ktalon
+description=Email quoted message striping re-purposed from talon
+version=1.4.9
+author=Sherub Thakur
+author_email=sherub.thakur@kayako.com
+url=https://github.com/kayako/ktalon
```

Comment:
It looks not good.

Reply:
Honesty is definitely not going to be @sherubthakur cause of death :)

Reply:
Maybe it is not the cause of death, but I consider it unnecessary to change the address of the project. The current address of the project is quite good.

Reply:
@ad-m @horkhe This PR was intended for the main mailgun repo. I could give reasoning to this but it will all sound like excuses. So I will just simply apologise.